TSPL.docc/LanguageGuide/Protocols.md

```diff
@@ -1368,6 +1421,92 @@ for level in levels.sorted() {
   ```
 -->
 
+## Implicit Conformance to a Protocol
+
+Some protocols are so common that you would write them on almost every type.
+For the following protocols,
+Swift automatically infers the conformance
+when you define a type that implements the protocol's requirements:
+
+- `Codable`
+- `Copyable`
+- `Sendable`
+- `BitwiseCopyable`
+
+[`Codable`]: https://developer.apple.com/documentation/swift/codable
+<!--
+The remaining definitions for the links in this list
+are in the section above, Protocols That Don't Have Requirements.
+-->
+
+You can still write the conformance explicitly,
+but it doesn't have any effect.
+To suppress an implicit conformance,
+write a tilde (`~`) before the protocol name in the conformance list:
```

Comment:
... especially since I tried to suppress Codable conformance, but the compiler complains about `Type 'Codable' (aka 'Decodable & Encodable') cannot be suppressed`